Core/HLE/HLE.cpp

```diff
@@ -626,7 +626,7 @@ size_t hleFormatLogArgs(char *message, size_t sz, const char *argmask) {
 			break;
 
 		case 'i':
-			APPEND_FMT("%d", regval);
+			APPEND_FMT("%u", regval);
```

Comment:
regval may just as well represent a sign as an unsigned value, there's no way to know. %d is as good as any value, just cast to int for clarity. That is:

    APPEND_FMT("%d", (int)regval);

I think the rest of the changes to %u are also unnecessary, and if anything should be done, it's this same thing.